packages/web-components/src/common/components/modal.svelte

```diff
@@ -17,12 +18,78 @@
     background: #00000052;
     z-index: 1;
   }
+  @media only screen and (min-width: 48em) {
+    .reapit-modal-wrapper .reapit-modal {
+      top: 50% !important;
```

Comment:
pls avoid to use !important because it's very hard to debug

Reply:
I'm updated